### ISSUE TITLE
Using autoslim to pretrain the standard supernet, but model's input is str rather than tensor.[Bug]

### ISSUE THREAD

**Issue by JunDaLee** [open-mmlab/mmrazor#378]
### Describe the bug

Using autoslim to pretrain the standard supernet, but model's input is str rather tensor. 

### To Reproduce

The command you executed.

```shell
python ./tools/train.py  \
./configs/pruning/mmcls/autoslim/autoslim_mbv2_1.5x_supernet_8xb256_in1k.py  \
--work-dir ./work_dir_autoslim/mbv2/

```

### Post related information

1. The output of `pip list | grep "mmcv\|mmrazor\|^torch"`
```python
mmcv               2.0.0rc2
mmrazor            1.0.0rc0       /home/li/mmrazor-1.x
torch              1.10.0+cu111
torchaudio         0.10.0+rocm4.1
torchvision        0.11.0+cu111
```

4. Your train log file if you meet the problem during training.

```python
Traceback (most recent call last):
  File "./tools/train.py", line 66, in <module>
    main()
  File "./tools/train.py", line 62, in main
    runner.train()
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmengine/runner/runner.py", line 1658, in train
    model = self.train_loop.run()  # type: ignore
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmengine/runner/loops.py", line 90, in run
    self.run_epoch()
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmengine/runner/loops.py", line 106, in run_epoch
    self.run_iter(idx, data_batch)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmengine/runner/loops.py", line 122, in run_iter
    outputs = self.runner.model.train_step(
  File "/home/li/mmrazor-1.x/mmrazor/models/algorithms/nas/autoslim.py", line 118, in train_step
    max_subnet_losses = self(batch_inputs, data_samples, mode='loss')
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/li/mmrazor-1.x/mmrazor/models/algorithms/base.py", line 132, in forward
    return self.loss(inputs, data_samples)
  File "/home/li/mmrazor-1.x/mmrazor/models/algorithms/base.py", line 147, in loss
    return self.architecture(inputs, data_samples, mode='loss')
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmcls/models/classifiers/image.py", line 110, in forward
    return self.loss(inputs, data_samples)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmcls/models/classifiers/image.py", line 219, in loss
    feats = self.extract_feat(inputs)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmcls/models/classifiers/image.py", line 192, in extract_feat
    x = self.backbone(inputs)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmcls/models/backbones/mobilenet_v2.py", line 237, in forward
    x = self.conv1(x)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/mmcv/cnn/bricks/conv_module.py", line 207, in forward
    x = self.conv(x)
  File "/home/li/anaconda3/envs/mmyolo2.0/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1102, in _call_impl
    return forward_call(*input, **kwargs)
  File "/home/li/mmrazor-1.x/mmrazor/models/architectures/dynamic_ops/default_dynamic_ops.py", line 63, in forward
    return F.conv2d(input, weight, bias, self.stride, self.padding,
TypeError: conv2d() received an invalid combination of arguments - got (str, Tensor, NoneType, tuple, tuple, tuple, int), but expected one of:
 * (Tensor input, Tensor weight, Tensor bias, tuple of ints stride, tuple of ints padding, tuple of ints dilation, int groups)
      didn't match because some of the arguments have invalid types: (str, Tensor, NoneType, tuple, tuple, tuple, int)
 * (Tensor input, Tensor weight, Tensor bias, tuple of ints stride, str padding, tuple of ints dilation, int groups)
      didn't match because some of the arguments have invalid types: (str, Tensor, NoneType, tuple, tuple, tuple, int)

```

### Additional context

\[here\]

**Comment by JunDaLee** [comment]
Yes. I can run by replacing mmrazor v1.0.0rc0 with v1.0.0rc1. Thank you.

**Comment by JunDaLee** [comment]
In addition, I don't fiind a search config from `configs/nas/mmcls/autoslim` , how to search for subnet on the trained supernet ?

**Comment by LKJacky** [comment]
Search config will be released in mmrazor v1.0.0rc2. It will come soon in the next several days.
